Makefile

```diff
@@ -94,6 +94,15 @@ else
 	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoe2e" -ginkgo.succinct
 endif
 
+# Run json e2e tests
+.PHONY: test-json-e2e
```

Comment:
```
test-json-e2e -> test-odo-json-formated-output
```
Does this make sense ? name looks precise to me

Reply:
sure :+1: 
sounds good to me